src/styles/episodic/site-base.css

```
@@ -0,0 +1,285 @@
+ * the palette as the daisyUI "episodic" theme, the brand tokens under
+ * @theme. What remains here is everything a colour token cannot carry -
+ * the label and tracking scales, the spacing ramp, the reading measure,
+ * and the element-level defaults the subsite was written against.
+ *
+ * Tailwind's preflight is deliberately not imported: these defaults are the
+ * reset this site was designed on, and preflight would strip the list
+ * markers and heading rhythm the content pages rely on.
+ */
+
+@font-face {
+  font-family: "Economica";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Economica" (font-family-name-quotes)

(font-family-name-quotes)